index.html

```diff
@@ -0,0 +1,136 @@
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+     
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+      
+    </div>
+  </nav>
+  </header>
+  <!--end of nav -->
+
+  <!--start of  section-->
+  <section class="section-wrapper">
+    <div class="section-header">
+      <h1>I'm Abdullah <br> Glad to see <br> you!</h1>
+    </div>
+    <div class="content-of-section">
+      <p>I'm a software developer!<br> I can help you build a product feature or a website. 
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me.
+      </p>
+    </div>
+    
+    <div class="social-icons">
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
+    </div>
+
+  </section> 
+      <!--work section-->
+      <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/Portfolio.png" alt="">
+        </div>
+        <div class="card-header">
+          <h3>Multi-post <br> stories</h3>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Back End Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2015</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
+        </div>
+      </section>
+      <!--end of work section-->
+
+      <!--end about me section-->
+     <section class="about_me">
+        <div class="about_container">
+          <h3 class="about_me_title"> About <br>Myself</h3>
+          <div class="about_info">
+            <h3 class="about_me_title-desktop">About me</h3>
+            <div class="about_description">
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
```

Comment:
- [ ] Kindly make sure the design in conforming the Figma mockup
- [ ] Kindly separate the see project button and the about section
<img width="435" alt="Screen Shot 2022-04-27 at 13 30 47" src="https://user-images.githubusercontent.com/28307960/165509158-5caf6f9e-e902-435c-8af2-24c23306a89f.png">
- [ ] Kindly add at least 4 projects